charts/incubator/minecraft-bungeecord/questions.yaml

```diff
@@ -202,7 +202,7 @@ questions:
                                   description: "The internal(!) port on the container the Application runs on"
                                   schema:
                                     type: int
-                                    default: 25578
+                                    default: 25575
```

Comment:
https://github.com/truecharts/charts/blob/cc43c6860e45a384a352fb58247e33f02c74275a/charts/incubator/minecraft-bungeecord/values.yaml#L30

`targetPort` is whatever `port` is set to

If not, then the env does not work

Reply:
With the env you change the listen port inside the container

Reply:
I see what you mean now, basically we should manually set it to the default value then?

Reply:
internal port == RCON_PORT env == `port`

`targetPort` should be == `port`